README.md

```diff
@@ -32,6 +32,15 @@ python -m pip install "napari[all]"
 
 If you prefer conda over pip, you can replace the last line with: `conda install -c conda-forge napari`
 
+### containers:
+Docker
+```sh
+docker pull ghcr.io/truatpasteurdotfr/napari:main
```

Comment:
with this PR, does one still need to use your branch?  Or is that something you used in testing that can be changed now?  Not sure I love the default container source being outside of the napari org